README.md

```diff
@@ -18,17 +18,17 @@ This Project serves the backend APIs required for [Real Dev Squad](https://reald
 
 ## Running the Project
 ```shell
-$ npm install
-$ npm start
+npm install
+node server.js
```

Comment:
Why add this command?

Reply:
We already have added `npm run dev` for dev purpose.
We should keep `npm start` as it is in the README.

Reply:
We should remove this from the 'Running the development' version.

We should add a section that says, "What happens in Production":
1. Install packages
2. Run build
3. Prune dev dependencies
4. Run start command (with port information)

Reply:
These changes have been merged via https://github.com/Real-Dev-Squad/website-backend/pull/513.
@HaroldPetersInskipp , can you update the PR as per the comments added here?